test/lib/aggregate_test_builder_test.dart

```diff
@@ -71,21 +71,69 @@ void main() {
         'a|test/templates/foo_template.html': '',
         'a|test/templates/bar_template.html': '',
         'a|test/vm_test.dart': "@TestOn('vm') library vm_test;",
-        'a|test/other_test.dart': '',
+        'a|test/a_test.dart': '',
+        'a|test/b_test.dart': '',
       }, outputs: {
         'a|test/templates/default_template.browser_aggregate_test.dart':
             '''@TestOn('browser')
 import 'package:test/test.dart';
 
-import '../other_test.dart' as other_test;
+import '../a_test.dart' as a_test;
+import '../b_test.dart' as b_test;
 
 void main() {
-  other_test.main();
+  a_test.main();
+  b_test.main();
 }
 '''
       });
     });
 
+    test('generates a default randomizes aggregate test for browser tests',
+        () async {
+      final config = TestHtmlBuilderConfig(
+          browserAggregation: true,
+          randomizeAggregation: true,
+          testShuffleSeed: 2,
+          templates: {
+            'test/templates/foo_template.html': ['test/foo_test.dart'],
+            'test/templates/bar_template.html': ['test/bar_test.dart'],
+          });
```

Comment:
We can probably omit the templates from this test since they're not even matching any of the tests below and this test doesn't really need to verify anything related to the template matching.